controller_manager/src/controller_manager.cpp

```diff
@@ -1485,7 +1485,7 @@ void ControllerManager::list_hardware_components_srv_cb(
     auto component = controller_manager_msgs::msg::HardwareComponentState();
     component.name = component_name;
     component.type = component_info.type;
-    component.class_type = component_info.class_type;
+    component.plugin_name = component_info.plugin_name;
```

Comment:
Can we deprecate this first? For not until ROS Iron?

Reply:
```suggestion
    component.class_type = component_info.plugin_name; // TODO: deprecated currently. Remove soon
    RCLCPP_WARN(get_logger(), "The 'class_type' field in 'controller_manager_msgs/msg/HardwareComponentState.msg' is deprecated and will be removed soon. Please switch over client code to use 'plugin_name' instead.");
    component.plugin_name = component_info.plugin_name;
```